### PR TITLE
Add Samples for Connected Clusters support

### PR DESCRIPTION
**Pull request by prasannajsns**
## Purpose
Add samples for connected cluster support. Two scenarios are covered - 1) ConnectedClusters-only 2) Connected and Managed Clusters
* ...

## Does this introduce a breaking change?
```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: samples/k8s-connected-and-managed-cluster/README.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+The following changes are required to enable support for connected and managed clusters.
+
+### Manifest
+
+Add 'supportedClusterTypes' under extensionRegistrationParameters.
+
+Sample below indicates all distributions of connectedClusters type and all ditributions of managedClusters type are supported.
+
+![Alt text](images/Manifest_changes.PNG)
```
Comment (thomasyip-msft) [inline samples/k8s-connected-and-managed-cluster/README.md]: It seems there are casing error as the image don't show well. Can you fix those? png vs PNG.

File: samples/k8s-connected-and-managed-cluster/README.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+Following is the guidance on the supportedClusterTypes:
+
+| Property      | Description |
+| -----------   | ----------- |
+| supportedClusterTypes       | Contains an object for each top-level cluster-type. Allowed types are – “managedClusters”, “connectedClusters”. "managedClusters" denotes Azure Kubernetes Service (AKS) clusters. "connectedClusters" denotes Azure Arc-enabled Kubernetes clusters. For each of these cluster types, specify distributions and unsupported Kubernetes versions for these distributions. If supportedClusterTypes is not provided, all distributions of ‘managedClusters’ will be supported by default. If supportedClusterTypes is provided, and a given top level cluster type is not provided, then all distributions and Kubernetes versions for that cluster type will be treated as unsupported. |
```
Comment (thomasyip-msft) [inline samples/k8s-connected-and-managed-cluster/README.md:36]: There are so much detail in this table, it seems the table format doesn't look that good.

File: samples/k8s-connected-and-managed-cluster/README.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+| distributionSupported  | A boolean value representing whether the specified distribution(s) are supported. If false, providing a value for UnsupportedVersions will cause an error. |
+| unsupportedVersions  | A list of versions for the specified distribution(s) which are unsupported. Supported operators: |
+
+                        •	"=" Given version is not supported. E.g.: “=1.2.12”
+
+                        •	">" All versions greater than the given version are not supported. E.g.: “>1.1.13”
+
+                        •	"<" All versions less than the given version are not supported. E.g.: “<1.3.14”
+
+                        •	"..." All versions in range are unsupported. E.g.: ” 1.1.2...1.1.15”  (includes right-side value and excludes left-side value)
```
Comment (thomasyip-msft) [inline samples/k8s-connected-and-managed-cluster/README.md:47]: Is this list supposed to be inside the table of unsupported versions?

File: samples/k8s-connected-and-managed-cluster/README.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+Only the "AKS" distribution of connectedClusters type is supported with restriction on some versions. Only the "AKS" distribution of managedClusters is supported with restriction on some versions.
+
+``` yaml
+  supportedClusterTypes:
+    connectedClusters:
+      - distribution: ["AKS"]
+        distributionSupported: true
+        unsupportedVersions: ["1.10.0...1.20.0", ">1.21.1"]
+    managedClusters:
+      - distribution: ["AKS"]
```
Comment (thomasyip-msft) [inline samples/k8s-connected-and-managed-cluster/README.md:73]: What is really the difference in AKS vs All in managedClusters?

File: samples/k8s-connected-and-managed-cluster/createUIDefinition.json [hunk 5]
```diff
@@ -0,0 +1,345 @@
...
+								"toolTip": "The SKU of Linux OS for VM.",
+								"constraints": {
+									"allowedValues": [
+										{
+											"label": "Ubuntu",
+											"value": "Ubuntu"
+										},
+										{
+											"label": "AzureLinux",
+											"value": "AzureLinux"
```
Comment (thomasyip-msft) [inline samples/k8s-connected-and-managed-cluster/createUIDefinition.json:231]: Can you set this as default?

File: samples/k8s-connected-and-managed-cluster/README.md (hunk 2 above)
Comment (prasannajsns) [reply samples/k8s-connected-and-managed-cluster/README.md:36]: Table does not seem to offer much formatting. For now, let's go with this and consider other ways of organizing this.

File: samples/k8s-connected-and-managed-cluster/README.md (hunk 4 above)
Comment (prasannajsns) [reply samples/k8s-connected-and-managed-cluster/README.md:73]: Both are same at this time as we don't have any other flavor of managedClusters.

File: samples/k8s-connected-and-managed-cluster/README.md (hunk 3 above)
Comment (prasannajsns) [reply samples/k8s-connected-and-managed-cluster/README.md:47]: Yes, this is supposed to be inside the previous cell, but table does not allow multi-lines. I added some text that indicates this list is linked to the previous row.

File: samples/k8s-connected-and-managed-cluster/createUIDefinition.json (hunk 5 above)
Comment (prasannajsns) [reply samples/k8s-connected-and-managed-cluster/createUIDefinition.json:231]: done

File: samples/k8s-connected-and-managed-cluster/README.md (hunk 1 above)
Comment (prasannajsns) [reply samples/k8s-connected-and-managed-cluster/README.md]: Fixed the casing mismatches.

**Review by thomasyip-msft — APPROVED** [review]

**Review by Nickomang — APPROVED** [review]

**Pull request merged**